contrib/kind.sh

```diff
@@ -46,7 +45,11 @@ usage() {
     echo "                 [-nbl|--ovn-loglevel-nb <loglevel>] [-sbl|--ovn-loglevel-sb <loglevel>]"
     echo "                 [-cl |--ovn-loglevel-controller <loglevel>] [-dl|--ovn-loglevel-nbctld <loglevel>]"
     echo "                 [-ep |--experimental-provider <name>] |"
-    echo "                 [-eb |--egress-gw-separate-bridge]"
+    echo "                 [-eb |--egress-gw-separate-bridge] |"
```

Comment:
what is with the "|" pipe after some of the options here?

Reply:
I was just doing this for mainly for formatting consistency, I'm not sure it serves any purpose